specs/signature-envelope-jws.md

```diff
@@ -1,18 +1,22 @@
 # JWS Signature Envelope
 
-This specification implements the [Notary v2 Signature specification](signature-specification.md) using JSON Web Signature (JWS). JWS ([RFC7515](https://datatracker.ietf.org/doc/html/rfc7515)) is a JSON based envelope format for digital signatures over any type of payload (e.g. JSON, binary). JWS is a Notary v2 supported signature format and specifically uses the *JWS JSON Serialization* representation.
+This specification implements the [Notary Project Signature specification](signature-specification.md) using JSON Web Signature (JWS). JWS ([RFC7515](https://datatracker.ietf.org/doc/html/rfc7515)) is a JSON based envelope format for digital signatures over any type of payload (e.g. JSON, binary). JWS is a Notary Project supported signature format and specifically uses the *JWS JSON Serialization* representation.
 
 ## Storage
 
-A JWS signature envelope will be stored in an OCI registry as a blob, and referenced in the signature manifest as a blob with `mediaType` of `"application/jose+json"`.
+A JWS signature envelope will be stored in an OCI registry as a layer, and referenced in the signature manifest as a layer with `mediaType` of `"application/jose+json"`.
 
 Signature Manifest Example
 
 ```jsonc
 {
     "mediaType": "application/vnd.oci.artifact.manifest.v1+json",
-    "artifactType": "application/vnd.cncf.notary.signature",
-    "blobs": [
+    "config": {
```

Comment:
Same question as above about which config mediaType to use.

Reply:
As answered in previous comment